### PR TITLE
Publishes Calico Cloud 20.3.1

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud_versioned_docs/version-20-2/release-notes/index.mdx [hunk 1]
```diff
@@ -5,6 +5,16 @@ title: Release notes
 
 # Calico Cloud release notes
 
+## December 19, 2024 (version 20.3.1)
+
+### Bug fixes
+
+* Previously, updating Calico Cloud introduced breaking changes for some users who had defined network policy tiers with orders higher than the default value.
+  This was caused by a change to the default tier order from 1,000 to 1,000,000.
```
Comment (ctauchen) [inline calico-cloud_versioned_docs/version-20-2/release-notes/index.mdx]: @tmjd I got the 1000 figure from the docs. Is that right?

Comment (tmjd) [reply calico-cloud_versioned_docs/version-20-2/release-notes/index.mdx]: No that's not right, it went from undefined to 1,000,000

File: src/___new___/data/ccImageLists.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const ccImageLists = {
-  // curl -0 https://installer.calicocloud.io/manifests/v3.20.0-2.0-9/image-list
-  'v20.3.0 (latest)': `quay.io/tigera/cnx-apiserver:v3.20.0-2.0
+  // curl -0 https://installer.calicocloud.io/manifests/v3.20.0-2.0-10/image-list
+  'v20.3.1 (latest)': `quay.io/tigera/cnx-apiserver:v3.20.0-2.0
+quay.io/tigera/cnx-apiserver:v3.20.0-2.0
```
Comment (tmjd) [inline src/___new___/data/ccImageLists.js]: This image is listed twice

**Review by tmjd — APPROVED** [review]
LGTM

**Pull request merged**